web/src/beta/components/Resizable/hooks.ts

```diff
@@ -142,9 +180,28 @@ export default (
   );
 
   const handleResetSize = useCallback(() => {
+    if (!parsedState || !localStorageKey) return;
+    localStorage.setItem(
+      localStorageKey,
+      JSON.stringify({
+        [localStorageKey]: {
+          size: initialSize,
+          minimized: false,
+          difference: 0,
+          startingSize: initialSize,
+        },
+      }),
+    );
```

Comment:
```suggestion
    if (localStorageKey) {
       localStorage.setItem(
         localStorageKey,
         JSON.stringify({
           [localStorageKey]: {
             size: initialSize,
             minimized: false,
             difference: 0,
             startingSize: initialSize,
           },
         }),
    )};
```

Reply:
Seems like parsedStated isn't needed in this callback, right?
Also, I think we should allow general functionality of Resizable to still work even if there is an issue/bug around the local storage. Thats why you should wrap the localStorage use, rather than returning early and not allowing anything to happen.